### PR TITLE
adds specific guidance for testing keyboard access on mac

### PR DESCRIPTION
**Pull request by cptvitamin**
@catong and @lamagnifica, can you take a look at this Note I added to the Developer guidelines for accessibility?

### CODE REVIEW

File: en_us/developers/source/accessibility.rst [hunk 1]
```diff
@@ -538,6 +538,14 @@ These are some automated tools for accessibility testing.
 
 * Your keyboard. For information about using your keyboard to test for
   accessibility, see `<http://webaim.org/techniques/keyboard/>`_.
+  
+.. note:: By default, the Mac OSX operating system is configured to move keyboard
+   focus to "Text boxes and lists only."  This setting also applies to browsing web
+   pages using Safari or Firefox with a keyboard.  To effectively test 
+   keyboard accessibility using a Mac, you should configure your computer
+   to focus on "All controls."  Open up your System Preferences -> Keyboard then
```
Comment (lamagnifica) [inline en_us/developers/source/accessibility.rst]: Reword (I love the convention of using > for a series of instructions, but we don't use it.)
I suggest, 
"Open **System Preferences**, and then select **Keyboard**. On the **Shortcuts** tab, for Full Keyboard Access select **All controls**. You can also toggle this setting with Control+F7."

**Comment by lamagnifica** [comment]
a suggestion for rewording a procedure. Otherwise, :+1:

**Comment by cptvitamin** [comment]
@lamagnifica one more look?  I incorporated your suggestion but also added new text.

File: en_us/developers/source/accessibility.rst [hunk 2]
```diff
@@ -538,6 +538,19 @@ These are some automated tools for accessibility testing.
...
 * Your keyboard. For information about using your keyboard to test for
   accessibility, see `<http://webaim.org/techniques/keyboard/>`_.
+  
+.. note:: By default, the Mac OSX operating system is configured to move keyboard
+   focus to **Text boxes and lists only**.  This setting also applies to browsing web
+   pages using Safari or Firefox with a keyboard.  To effectively test 
+   keyboard accessibility using a Mac, you should configure your computer
+   to focus on **All controls**.  Open **System Preferences**, and then select **Keyboard**. 
+   On the **Shortcuts** tab, for Full Keyboard Access select **All controls**. You can also 
+   toggle this setting with Control+F7.  If you are a Chrome user, this behavior is
```
Comment (lamagnifica) [inline en_us/developers/source/accessibility.rst]: maybe a paragraph break after the F7 sentence? just continue to indent 3 spaces and the whole thing will be in a colored note box

File: en_us/developers/source/accessibility.rst [hunk 3]
```diff
@@ -538,6 +538,19 @@ These are some automated tools for accessibility testing.
...
+  
+.. note:: By default, the Mac OSX operating system is configured to move keyboard
+   focus to **Text boxes and lists only**.  This setting also applies to browsing web
+   pages using Safari or Firefox with a keyboard.  To effectively test 
+   keyboard accessibility using a Mac, you should configure your computer
+   to focus on **All controls**.  Open **System Preferences**, and then select **Keyboard**. 
+   On the **Shortcuts** tab, for Full Keyboard Access select **All controls**. You can also 
+   toggle this setting with Control+F7.  If you are a Chrome user, this behavior is 
+   controlled in a browser setting and is enabled by default.  However, if you find that you
+   cannot move focus to links while using Chrome you may need to change your browser configuration.
```
Comment (lamagnifica) [inline en_us/developers/source/accessibility.rst]: while using Chrome you may need to --> while using Chrome, you might need to

File: en_us/developers/source/accessibility.rst [hunk 4]
```diff
@@ -538,6 +538,19 @@ These are some automated tools for accessibility testing.
...
+   Open **Settings**, then click **Show advanced settings...**.  Confirm that the
```
Comment (lamagnifica) [inline en_us/developers/source/accessibility.rst]: pro tip: no need to include the punctuation (...) that appears in a UI label in the doc

File: en_us/developers/source/accessibility.rst [hunk 5]
```diff
@@ -538,6 +538,19 @@ These are some automated tools for accessibility testing.
...
+   pages using Safari or Firefox with a keyboard.  To effectively test 
+   keyboard accessibility using a Mac, you should configure your computer
+   to focus on **All controls**.  Open **System Preferences**, and then select **Keyboard**. 
+   On the **Shortcuts** tab, for Full Keyboard Access select **All controls**. You can also 
+   toggle this setting with Control+F7.  If you are a Chrome user, this behavior is 
+   controlled in a browser setting and is enabled by default.  However, if you find that you
+   cannot move focus to links while using Chrome you may need to change your browser configuration.
+   Open **Settings**, then click **Show advanced settings...**.  Confirm that the 
+   **Pressing Tab on a web page highlights links, as well as form fields** checkbox
+   under **Web content** is checked.
```
Comment (lamagnifica) [inline en_us/developers/source/accessibility.rst]: move "under **Web content** " to the beginning of the sentence: "Under **Web content**, confirm that... " (that is, place each action in the order that it actually has to be done)

checked --> selected

**Comment by lamagnifica** [comment]
a few nits, otherwise :+1:

**Pull request merged**